src/Evolution/DiscontinuousGalerkin/SlopeLimiters/WenoGridHelpers.cpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/DiscontinuousGalerkin/SlopeLimiters/WenoGridHelpers.hpp"
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"  // IWYU pragma: keep
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Mesh.hpp"  // IWYU pragma: keep
+#include "Domain/Side.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace SlopeLimiters {
+namespace Weno_detail {
+
+template <size_t VolumeDim>
+std::array<DataVector, VolumeDim> neighbor_grid_points_in_local_logical_coords(
+    const Mesh<VolumeDim>& neighbor_mesh,
+    const Direction<VolumeDim>& direction_to_neighbor) noexcept {
+  std::array<DataVector, VolumeDim> result{{{}}};
```

Comment:
maybe `result` -> `neighbor_coords` or something?

Reply:
I worry that `neighbor_coords` can be ambiguous about whether "neighbor" refers to the points or the coordinate system. I opted for (what I hope is) a very clear function name with something  that can't be misinterpreted for the variable name. I'm happy to rename if you can suggest something that isn't ambiguous 🙂